expenses/reports.py

```diff
@@ -118,14 +120,14 @@ def query(self, cursor, sql: str) -> typing.Iterable:
         cursor.execute(sql, sql_params)
         return cursor.fetchall()
 
-    def get_column_headers(self, engine: Engine) -> (typing.List[str], typing.List[str]):
+    def get_column_headers(self, engine: Engine, is_html=True) -> (typing.List[str], typing.List[str]):
         return self.column_headers
 
-    def preprocess_rows(self, results: typing.Iterable) -> typing.Iterable:
+    def preprocess_rows(self, results: typing.Iterable, is_html=True) -> typing.Iterable:
         return results
 
     def tabulate(self, results: typing.Iterable, engine: Engine) -> SafeString:
-        column_headers: (typing.List[str], typing.List[str]) = self.get_column_headers(engine)
+        column_headers: (typing.List[str], typing.List[str]) = self.get_column_headers(engine, False)
```

Comment:
`get_column_headers` should be called with the correct `is_html` value, depending on the output format.